packages/pricing/src/models/price-list.ts

```diff
@@ -59,33 +60,32 @@ export default class PriceList {
     columnType: "timestamptz",
     nullable: true,
   })
-  starts_at: Date | null
+  starts_at: Date | null = null
 
   @Property({
     columnType: "timestamptz",
     nullable: true,
   })
-  ends_at: Date | null
+  ends_at: Date | null = null
 
   @OneToMany(() => PriceSetMoneyAmount, (psma) => psma.price_list, {
-    cascade: [Cascade.REMOVE],
+    cascade: ["soft-remove" as Cascade],
   })
   price_set_money_amounts = new Collection<PriceSetMoneyAmount>(this)
 
   @OneToMany(() => PriceListRule, (pr) => pr.price_list, {
-    cascade: [Cascade.REMOVE],
+    cascade: ["soft-remove" as Cascade],
   })
   price_list_rules = new Collection<PriceListRule>(this)
 
   @ManyToMany({
     entity: () => RuleType,
     pivotEntity: () => PriceListRule,
-    cascade: [Cascade.REMOVE],
```

Comment:
nice 🙏 , just to be aware, Cascade.REMOVE on collection will perform one query per item under the hood if used